pkg/commands/workload_get.go

```diff
@@ -114,6 +114,16 @@ func (opts *WorkloadGetOptions) Exec(ctx context.Context, c *cli.Config) error {
 
 	c.Printf(printer.ResourceStatus(workload.Name, printer.FindCondition(workload.Status.Conditions, cartov1alpha1.WorkloadConditionReady)))
 
+	if workload.Status.SupplyChainRef == (cartov1alpha1.ObjectReference{}) && len(workload.Status.Conditions) == 0 {
```

Comment:
There four cases, as far as I think
1. When there are `conditions`, but no `SupplyChainRef`
2. When there is `SupplyChainRef`, but no `conditions` (this is very unlikely to happen)
3. When there are neither of both
4. When both of them are present

This check is to validate the third case is covered. As spoken with design, if there are `conditions` information in `workload.status.conditions`, even if there is no `SupplyChainRef`, the data for `last update` and `ready` rows is going to be present and this can be shown. Same the other way around, if there are no `conditions` but there is `SupplyChainRef`, yet there is a row (`name`) that can show some info. The only way we cannot display any info is when we do not have any of those, so we show a message indicating this.

Yet, if you still think this is a no needed validation, I can remove it. I just thought I should take care of these possible cases.